examples/app-with-state/tsconfig.json

```diff
@@ -17,5 +17,6 @@
         "noFallthroughCasesInSwitch": true
     },
     "include": ["src"],
+    "exclude": ["dist"],
```

Comment:
This is not needed as it's never included.

Reply:
yeeh its weird, i again had some issues here with files in the dist folder, ill try it again without to see if the error still is there.